source/conf.py

```diff
@@ -80,6 +81,8 @@
 copybutton_prompt_text = r">>> |\.\.\. |\$ |In \[\d*\]: | {2,5}\.\.\.: | {5,8}: "
 copybutton_prompt_is_regexp = True
 
+suppress_warnings = ["myst.header", "myst.nested_header"]
```

Comment:
TIL you can do this. These warnings were why I hadn't bothered with `-W` before.